### PR TITLE
Update speaker and workshop description

### PR DESCRIPTION
**Pull request by andrazhribernik**
This PR contains updated speakers' bio and workshop description.

### CODE REVIEW

File: sessions/workshops/natural-language-processing-in-10-lines-of-code.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-
-In this workshop we will answer these and similar questions using the open source library spaCy. One hour spent with us will be useful for anybody who is interested in text processing, text information extraction or text related data science. You do not need to have prior experiences with NLP or data science. However, some Python programming skills are needed. We would also encourage you to install spaCy in advance (link to Github repo with installation instructions).
\ No newline at end of file
+Do you know that there are over 150 personal names mentioned in Jane Austen's Pride & Prejudice? Did you ever want to remove common words such as 'we', 'what', and 'for' from text? Or how to extract interesting keywords from a document? Did you know all of this could be done in 10 lines of Python code?
+In this workshop we will answer these and similar questions using the open source library [spaCy](http://spacy.io/). One hour spent with us will be useful for anybody who is interested in text processing, text information extraction or text related data science. You will leave with the tools and knowledge to start developing your own natural language processing projects in Python.
```
Comment (alexwlchan) [inline sessions/workshops/natural-language-processing-in-10-lines-of-code.md]: It looks like you’re trying to start a new paragraph – if so, you’ll need to add an extra blank line between the paragraphs.

Comment (alexwlchan) [inline sessions/workshops/natural-language-processing-in-10-lines-of-code.md:6]: Did you mean for this to be split across different paragraphs? Right now it all runs into one – you’ll need to add empty lines to get paragraph breaks.

Comment (andrazhribernik) [reply sessions/workshops/natural-language-processing-in-10-lines-of-code.md:6]: Good spot. Will update it.

File: speakers/andraz-hribernik.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
 name: Andraz Hribernik
 ---
 Andraz Hribernik
-Andraz is Data Engineer at Cytora. He is interested in big data processing, best engineering practices and ML. More specifically, he is trying to build processing pipeline that combines various advanced algorithms to be as easy to use as sklearn library. Before joining Cytora Andraz earned MSc in Computer Science, did research internship at Stanford University and worked for two successful Slovenian startups (Zemanta, Celtra).
-
+- Andraz is Data Engineer at Cytora. He is interested in big data processing, engineering best practices and ML. More specifically, his aim is to build a processing pipeline that combines various advanced algorithms to be as easy to use as the sklearn library. Before joining Cytora, Andraz earned an MSc in Computer Science, did a research internship at Stanford University and worked for two successful Slovenian startups (Zemanta, Celtra).
```
Comment (alexwlchan) [inline speakers/andraz-hribernik.md:5]: Similarly, if you want these to show up as bullets, you need to add a newline before and after the lines beginning with a hyphen.

**Comment by alexwlchan** [comment]
Content looks fine, just two formatting suggestions to make sure it looks good on the page.

**Comment by andrazhribernik** [comment]
@alexwlchan updated

**Comment by alexwlchan** [comment]
Thanks @andrazhribernik!

**Pull request merged**